contracts/l2/governance/l2_governance_relay.cairo

```diff
@@ -43,8 +38,15 @@ func relay{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr}(
 ) {
     let (l1_governance_relay) = _l1_governance_relay.read();
     assert l1_governance_relay = from_address;
-
-    ISpell.delegate_execute(spell);
+     
+    const delegate_execute_selector=1715357134534920869852627606170305435965756153030215653526748248853578673782;
```

Comment:
could you add a comment to explain from where this value comes from?

Reply:
and use capital letters for this constant :)

Reply:
sure